include/libtorrent/file_storage.hpp

```diff
@@ -235,7 +235,7 @@ namespace aux {
 		// tree.
 		static constexpr std::int64_t max_file_size = std::min(
```

Comment:
We need to defend this one too: `static constexpr std::int64_t max_file_size = (std::min)(`

Reply:
After that, I can confirm it works in a x86_64 context on MSVC (haven't tested x86 but shouldn't be an issue either).